src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -1426,23 +1426,51 @@ class LedgerMasterImp
         if (!referenceLedger || (referenceLedger->getLedgerSeq() < index))
             return ledgerHash; // Nothing we can do. No validated ledger.
 
-        // See if the hash for the ledger we need is in the reference ledger
+        // Is the hash for the ledger we need in the reference ledger?
         ledgerHash = referenceLedger->getLedgerHash (index);
-        if (ledgerHash.isZero())
+
+        // Yes
```

Comment:
You can probably remove line 1432, it assumes a pretty poor attention span on the reader.